src/assets/scss/_desktop_navigation.scss

```diff
@@ -88,6 +88,14 @@ $promo-width: 200px;
       background: $colorGrayscale-pebble;
       display: block;
     }
+    > li:nth-of-type(5) {
```

Comment:
As it is, yes, it should be a fixed 5 as the Give link will be the last list item. However, @riceguitar and I have talked about alternate solutions since then that I'm working on now.